package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "podio-js",
-  "version": "1.4.0",
+  "version": "1.4.2",
```

Comment:
I did use npm version. I'll remember to run it on master from now on, though. Thanks